hadoop/src/main/java/org/apache/iotdb/tsfile/hadoop/TSFInputSplit.java

```diff
@@ -38,7 +38,7 @@
  * Each <code>InputSplit</code> will be processed by individual
  * <code>Mapper</code> task.
  */
-public class TSFInputSplit extends InputSplit implements Writable {
+public class TSFInputSplit extends InputSplit implements Writable, org.apache.hadoop.mapred.InputSplit {
```

Comment:
Why not just user one `InputSplit`? 
`org.apache.hadoop.mapred.InputSplit` is the interface of hadoop1, `org.apache.hadoop.mapreduce.InputSplit` is the interface of hadoop2.
Hive-Connect must use the interface of hadoop1, but the Hadoop-connect can use the interface of hadoop2.

Reply:
I want to reuse the TSFInputSplit in Hadoop-Connector module, it uses the new API of Hadoop.
However Hive 2.3.6 still use the old one, so I just let the TSFInputSplit class implemented the old interface.
Thus, it can work both for hadoop-connector and hive-connector module